src/pages/send-tokens/components/send-max-button.tsx

```diff
@@ -33,10 +35,12 @@ interface SendMaxProps {
 
 export function SendMaxButton(props: SendMaxProps): JSX.Element | null {
   const { isLoadingFee, onClick } = props;
+  const { selectedAsset } = useSelectedAsset();
```

Comment:
Could this be passed in as a prop, to keep the component dumb?

Reply:
Yep, I can change it.